libs/trustchain/tests/scenarios/removedMemberEjectedOnDeletedTrustchain.ts

```diff
@@ -13,7 +13,7 @@ export async function scenario(transport: Transport, { sdkForName }: ScenarioOpt
   const member2 = { name: name2, id: member2creds.pubkey, permissions: 0xffffffff };
 
   // auth with the device and init the first trustchain
-  const { trustchain } = await sdk1.getOrCreateTrustchain(transport, member1creds);
+  const { trustchain } = await sdk1.getOrCreateTrustchain("foo", member1creds);
```

Comment:
could we just use `""` instead of foo? or maybe we can also make a deviceId constant somewhere to explicit what this is.

Reply:
I ended up passing the device id to the test scenario instead of the transport